app/src/main/java/com/google/android/samples/socialite/ui/home/Home.kt

```diff
@@ -38,38 +38,64 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
-import androidx.lifecycle.viewmodel.compose.viewModel
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
 import com.google.android.samples.socialite.R
 import com.google.android.samples.socialite.ui.AnimationConstants
 import com.google.android.samples.socialite.ui.home.timeline.Timeline
 
+@OptIn(ExperimentalMaterial3AdaptiveNavigationSuiteApi::class)
 @Composable
 fun Home(
     onChatClicked: (chatId: Long) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    var destination by rememberSaveable { mutableStateOf(Destination.Chats) }
+    val context = LocalContext.current
+    var currentDestination by rememberSaveable { mutableStateOf(Destination.Chats) }
+    NavigationSuiteScaffold(
+        navigationSuiteItems = {
+            for (destination in Destination.entries) {
+                val selected = currentDestination.route == destination.route
+                val label = context.getString(destination.label)
```

Comment:
I think this should be `stringResource` still:
```suggestion
                val label = stringResource(destination.label)
```

Reply:
I see "@Composable invocations can only happen from the context of a @Composable function" if I use stringResource

Reply:
Ah that's annyoing - `navigationSuiteItems` itself isn't composable. Using `context.getString` would have an edge-case bug where it wouldn't update the string used if the locale were to change without recreating the activity. We won't run into that right now (the manifest setup means we do recreate the activity on locale changes) but a more general way would probably be to try to resolve all the labels with `stringResource` outside of `navigationSuiteItems` to then pass into it.

Reply:
Oh a better solution! If you inline `label` into the `label` slot for `item`, then you can call `stringResource` there

Reply:
Oh good to know about that edge case with context.getString! Thank you. I've inlined stringResource within item.